test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol

```diff
@@ -176,6 +173,17 @@ contract ArbitraryRequesttsol is BaseCCRTest {
             hash // Hash of the VAA being used
         );
 
-        assertEq(pushNttToken.balanceOf(address(coreProxy)), amount);
+        assertEq(pushToken.balanceOf(address(coreProxy)), amount);
+    }
+
+    function test_when_UserTries_ClaimingArbitraryTokens() external {
```

Comment:
add a assert check for updated protocol_pool_fee as well
in all claiming test functions